cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -343,6 +344,11 @@ def export_handler(request, course_key_string):
     requested_format = request.GET.get('_accept', request.META.get('HTTP_ACCEPT', 'text/html'))
 
     if request.method == 'POST':
+        if not has_view_object_tags_access(request.user, str(course_key)):
+            raise PermissionDenied(
+                "You do not have permission to view object tags for this object_id."
```

Comment:
> We're already checking if not has_course_author_access(request.user, course_key): - isn't that enough?

I think you are right, I will follow this. :+1: 

> If you want to keep it in, we should just disable the tags.csv file when the user doesn't have permission, rather than making the whole export fail.

I think do this will do the code more complex

Reply:
Yeah, just take it out for now. The existing check for `has_course_author_access` is sufficient. If you're allowed to export the course, you're allowed to see all the tags.

Reply:
Updated [here](https://github.com/openedx/edx-platform/pull/34356/commits/a271fde7be93ebbb3f36b927f867e1e38d2c74f6#diff-04b2acdc3a891b60084044b36b19700dd497759339ead9397241c0b656b38d62L347)